test/gie/velocity.gie

```diff
@@ -0,0 +1,68 @@
+BEGIN
+===============================================================================
+Test for the velocity operation - Kinematic Gridshifting
+
+For all the velocity tests the alaska and egm96_15.gtx grids are used even
+though they are not parts of a velocity model, they are in the proper format
+and for testing purposes it doesn't really matter all that much...
+
+The input coordinate is located at lon=60, lam=-160 - somewhere in Alaska.
+
+===============================================================================
+
+TOLERANCE   0.000001 m
+
+-------------------------------------------------------------------------------
+Test using only horizontal grid and +tobs parameter
+-------------------------------------------------------------------------------
+OPERATION   +proj=velocity +xy_grids=alaska +t_epoch=2016.0 +t_obs=2000.0
+-------------------------------------------------------------------------------
+ACCEPT      -3004295.5882503074 -1093474.1690603832 5500477.1338251457
+EXPECT      -3004295.5888766116 -1093474.1688513425 5500477.1338251457
+ROUNDTRIP   1000
+
+-------------------------------------------------------------------------------
+Test using only vertical grid and +tobs parameter
+-------------------------------------------------------------------------------
+OPERATION   +proj=velocity +z_grids=egm96_15.gtx +t_epoch=2016.0 +t_obs=2000.0
```

Comment:
Does this make sense to use egm96_15.gtx ? This does not contain velocity values, unless I'm missing something. I can understand this is practical for testing, but some comment should mention this is not meant to be a real use case, just a technical one
(perhaps the same with the alaska grid ?)

Reply:
No. Except that it doesn't matter in a test setting. There's a comment about precisely that point in the top of the file. I don't want to clutter the repository with too many grid files, since we already have another way of distributing those.

Reply:
Ah sorry for missing the comment. I never read file headers...

Reply:
No worries.

The whole point with this stuff is to implement scandinavian transformations that uses a velocity model. When I've got this code sorted (there's some failing tests I need to fix), then I'll work on some actual data and hopefully confirm that this works as intended. Maybe I'll include that as test cases in the end.

Reply:
I am actually already using GDAL for that. A few weeks ago I wrote [this small utility](https://github.com/Kortforsyningen/kmsgrid/blob/master/kmsgrid.py) with the intend of translating various grids files from the Danish TrLib system to formats that PROJ.4 understands.

Good point regarding creating a specific test grid. I'll do that once I am happy with the code.